scripts/connect_character.sh

```diff
@@ -1,21 +1,26 @@
 #! /bin/bash
-SERVER=http://localhost:8001
-SECRET="abcdefgh"
+DEFAULT_SERVER=localhost
+DEFAULT_PORT=8001
```

Comment:
Oh! I didn't realise that the default port was 8001. However this is the one used by webpack-dev-server in `dilia` Could you change it to 8002? Alernatively, I can update the one used by webpack-dev-server. Please let me know if you want me to do the change.